Library/Homebrew/cmd/help.rb

```diff
@@ -74,6 +74,10 @@ def help(cmd = nil, flags = {})
   end
 
   def command_help(path)
+    if path.to_s.include? "Homebrew/cmd/"
```

Comment:
Ruby has a built in method for checking if a `Pathname` matches a given path pattern, which also would be a little more accurate than `String#include?`:

```ruby
Pathname(path).fnmatch?("Homebrew/cmd/*")
```

(The call to `Pathname()` will convert the argument to a `Pathname` instance if it isn't one already.)